outlay/app/build.gradle

```diff
@@ -144,6 +144,7 @@ dependencies {
         exclude group: 'com.android.support', module: 'support-annotations'
     }
     testCompile 'junit:junit:4.12'
+    testCompile group: 'org.mockito', name: 'mockito-all', version: '1.9.+'
```

Comment:
This could be in the more usual format of `testCompile "org.mockito:mockito-all:1.9.+"`. Additionally, using fixed versions instead of the `+` notation would be preferred. The current latest Mockito version is `2.8.18`.